autoload/easy_align.vim

```diff
@@ -89,9 +89,33 @@ function! s:floor2(v)
   return a:v % 2 == 0 ? a:v : a:v - 1
 endfunction
 
+function! s:get_highlight_group_name(line, col)
+  let l:hl = synIDattr(synID(a:line, a:col, 0), 'name')
+
+  if l:hl == '' && has('nvim')
+    " try getting the highlighting group using neovim treestiiter approach.
+	" The required functionality appeared in neovim nightly around 18 Dec, 2022
+	" https://www.reddit.com/r/neovim/comments/zou5ig/inspect_is_now_available_on_nightly/
+	let insp = luaeval( 'vim.inspect_pos and vim.inspect_pos( nil, ' .. (a:line-1) .. ', ' .. (a:col-1) .. ' ) or { treesitter = {} }' )
+    if ! empty( insp.treesitter )
```

Comment:
Is this an array? Then,

```suggestion
    if len(insp.treesitter)
```

Reply:
The help of the `empty()` function says:

```
Return the Number 1 if {expr} is empty, zero otherwise.
- A List or Dictionary is empty when it does not have any
  items.
```

Why exactly is it wrong to use it here?  Compatibility with old versions of vim?

Reply:
It's a style suggestion. I often use `len(something)` instead of `!empty(something)` because it's more concise, but using `!empty` is fine. But please remove extra spaces to match the code style.

```suggestion
    if !empty(insp.treesitter)
```